public/wp-content/themes/ColoredCow/functions.php

```diff
@@ -17,7 +16,9 @@ function cc_scripts() {
     function cc_styles() {  
         wp_enqueue_style('cc-bootstrap', get_template_directory_uri().'/dist/lib/css/bootstrap.min.css');
         wp_enqueue_style('cc-bootstrap-css','https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css');
-        wp_enqueue_style('cc-custom-font','https://fonts.googleapis.com/css?family=Oswald');
+        wp_enqueue_style('cc-custom-font1','https://fonts.googleapis.com/css?family=Oswald');
```

Comment:
Also, you can call multiple fonts within a single request. Here you are making 3 calls to google API and your page will take more time to load. Read on how you can load multiple fonts from google cdn in a single request.